amp/WEB-INF/src/org/digijava/kernel/ampapi/endpoints/indicator/manager/MEIndicatorDTO.java

```diff
@@ -58,7 +61,9 @@ public class MEIndicatorDTO {
     private AmpIndicatorGlobalValue targetValue;
 
     @JsonProperty("sectors")
+    @NotEmpty(message = "Sector list cannot be empty.")
```

Comment:
sector list can indeed be empty since you can tight an indicator only to a program

Reply:
![image](https://user-images.githubusercontent.com/29124635/226726392-7802b1b6-d831-4995-b34a-136ca7c3ea58.png)

@jdeanquin-dg i did that change based on lili's comments

Reply:
Remove the NotEmpty annotation